### DISCUSSION TITLE
Flashy animation artefact

### DISCUSSION THREAD

**Discussion by aluminum1** [keiwando/toonsquid#477] [General]
Hi - I love toonsquid. Trying to get my head around something. See below attachment for animation
I made (using 4 Bezier keyframes) of a planet going round an ellipse. Question: how come it has a "flashy"
behaviour? The edges of the planet seem to flash on and off in a jerky way. Is there a way to get perfectly smooth motion?

https://github.com/keiwando/toonsquid/assets/18632281/b97051af-2dc4-4adc-b53f-ec55a031b234

**Comment by keiwando** [comment]
Hi - thanks!
I've looked through the video frame by frame and am not exactly sure what kind of "flashy" behaviour you mean. The red dot looks the same to me on each frame of the video. Can you maybe post images of two frames where the edges seem to be flashing on and off?
